src/dune_engine/build_system.mli

```diff
@@ -189,10 +187,19 @@ module For_command_line : sig
   val eval_build_request : 'a Action_builder.t -> ('a * Dep.Set.t) Memo.Build.t
 end
 
-val get_cache : unit -> caching option
-
 val cache_teardown : unit -> unit
 
 (** {2 Running a build} *)
 
 val run : (unit -> 'a Memo.Build.t) -> 'a Fiber.t
+
+(** {2 Misc} *)
+
+module Progression : sig
```

Comment:
small nit: `Progress` sounds more natural to me.

Reply:
Sounds good!